### PR TITLE
feat(vscode): Create a vscode extension

### PR DESCRIPTION
**Pull request by mrmeku**

### CODE REVIEW

File: libs/server/src/lib/api/run-command.ts [hunk 1]
```diff
@@ -46,10 +42,14 @@ function createExecutableCommand(
   cmds: string[]
 ) {
   return () => {
-    const commandRunning = spawn(program, normalizeCommands(cwd, cmds), {
-      cwd,
-      cols: 80
-    });
+    const commandRunning = require('node-pty-prebuilt').spawn(
```
Comment (vsavkin) [inline libs/server/src/lib/api/run-command.ts]: why did you move the `require`?

File: libs/server/src/lib/api/install-nodejs.ts [hunk 2]
```diff
@@ -32,7 +31,7 @@ export function installNodeJs(): InstallNodeJsStatus {
   try {
     switch (platform()) {
       case 'darwin':
-        nodeDownloadProgress = progress({
+        nodeDownloadProgress = require('progress-stream')({
```
[the remark this answers is not in the document]
Comment (mrmeku) [reply libs/server/src/lib/api/install-nodejs.ts]: Required by tslint now that this file is linted vs ng lint

File: libs/server/src/lib/api/run-command.ts (hunk 1 above)
Comment (mrmeku) [reply libs/server/src/lib/api/run-command.ts]: Required by tslint now that this file is linted vs ng lint

File: libs/feature-workspaces/src/lib/workspace/workspace.component.ts [hunk 3]
```diff
@@ -168,17 +168,17 @@ export class WorkspaceComponent implements OnInit, OnDestroy {
...
-          this.routes[0],
-          {
-            icon: 'timeline',
-            url: 'connect/affected-projects',
-            title: 'Affected Projects'
-          },
-          ...this.routes.slice(1)
-        ];
-      }
+      // if (this.settings.isConnectUser()) {
```
[the remark this answers is not in the document]
Comment (mrmeku) [reply libs/feature-workspaces/src/lib/workspace/workspace.component.ts]: Should not have been. Undone now
